tests/yaml_parser_test.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+source ../bin/yaml.sh # provides parse_yaml and create_variables
+source ../bin/logging.sh # PSR-3 compliant logging
+
+CLUSTER_CONFIG_PATH='../.cluster.dev/minikube-one.yaml'
+
+for CLUSTER_MANIFEST_FILE in $(find "$CLUSTER_CONFIG_PATH" -type f  || ERROR "Manifest file/folder can't be found" && exit 1 ); do
```

Comment:
ERROR+ log levels run exit 1 by default.
https://github.com/shalb/cluster.dev/blob/master/docs/bash-logging.md#handlers

```suggestion
for CLUSTER_MANIFEST_FILE in $(find "$CLUSTER_CONFIG_PATH" -type f  || ERROR "Manifest file/folder can't be found"); do
```